Common/itkRecursiveBSplineInterpolationWeightFunction.h

```diff
@@ -75,9 +75,6 @@ class ITK_TEMPLATE_EXPORT RecursiveBSplineInterpolationWeightFunction
   using typename Superclass::SizeType;
   using typename Superclass::ContinuousIndexType;
 
-  /** The number of weights. */
-  static constexpr unsigned NumberOfWeights = (VSplineOrder + 1) * VSpaceDimension;
```

Comment:
For the record, originally it as defined by `GetConstNumberOfWeightsHackRecursiveBSplineType::Value`, here: https://github.com/SuperElastix/elastix/blob/5a80b720269e7c6b660283d75bb9a087f6dcb678/src/Common/itkRecursiveBSplineInterpolationWeightFunction.h#L112